modules/tester.js

```diff
@@ -1063,7 +1063,7 @@ Tester.prototype.begin = function begin() {
         config = getConfig([].slice.call(arguments)),
         next = function() {
             config.test(this, this.casper);
-            if (this.options.concise)
+            if (!this.options.concise)
```

Comment:
I think the merge could be done now, and the curly brackets consistency added later.